api/routes.go

```diff
@@ -94,4 +94,9 @@ func RegisterRoutes() {
 		"api-bsf",
 		shared.WrapApplicationJSON(shared.WrapPermissiveCORS(apiBSFHandler)),
 	)
+
+	// API endpoint for fetching test history
+	shared.AddRoute("/api/history", "api-history",
```

Comment:
I imagine this is boilerplate code for testing, but this will eventually need to take a test name to get the history of the specific test.